docs/how-to.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How-Tos
+order: 50
+---
+
+# Repository Changes
+
+**NOTE:** Changes should be coordinated such that the access controls at
+`hg.mozilla.org` are always more restrictive than those configured in
+mozilla-taskcluster.  For example, when raising a repo from level 2 to level 3,
+make the change in `hg.mozilla.org` first.  Likewise when lowering a repo from
+level 3 to level 2, make the change in mozilla-taskcluster first. This avoids a
+situation where a user with level-2 credentials could trigger level-3 tasks.
```

Comment:
Hm, I don't feel like that's a good approach.  Probably a project branch is a better strategy.